soc/xtensa/intel_adsp/ace/multiprocessing.c

```diff
@@ -96,6 +96,15 @@ void soc_start_core(int cpu_num)
 		DSPCS.bootctl[cpu_num].battr |= DSPBR_BATTR_LPSCTL_BATTR_SLAVE_CORE;
 	}
 
+	/* Disabling power before core power up. This step is required if core is starting after
+	 * entering power gating and being turn off.
+	 */
```

Comment:
shouldn't there be a "Disabling power **gating** before ..."

Reply:
yep, this comment is a little bit hard to understand

Reply:
This bit is not responsible for power gating. The comment has been updated.